README.md

```diff
@@ -139,6 +139,18 @@ A style guide for Sightly, the HTML templating system from Adobe Experience Mana
         ...
     </sly>
     ```
+    
+  - [3.4](#3.4) <a name='3.4'></a> **Always provide i18n translation hints.**
+  
+  The more context the translators have about a specific text, the better they can do their work.
```

Comment:
> I thought the hints were just placed as extra comment information. 
Not true, the hint/comment becomes part of the actual key! Therefore adding a hint to an already translated item, makes the translation no longer apply to that text.

-> Always provide i18n translation hints for non-descriptive text.
What is non-descriptive text?

I would rather leave the rule out, because it has some implications on how you should translate.
Also it causes more efforts for the translators because the entries for
`"${'Your topic' @ i18n, hint='some description.'}"`
`"${'Your topic' @ i18n, hint='some other description.'}"`
need to be translated separately!
Only if you have a very strict rule for naming your description and if you can live with the higher costs for translation that rule makes sense.